android/assets/jsons/translations/template.properties

```diff
@@ -228,7 +228,12 @@ Please don't spread your religion to us. =
 Don't spread religion in our cities. = 
 Very well, we shall spread our faith elsewhere. = 
 We noticed you have continued spreading your faith, despite your promise. This will have...consequences. = 
- 
+
+Please don't spy on us. = 
```

Comment:
"Stop spying on us"
We've had several users saying that the "please" format does not suit demands